### ISSUE TITLE
ToolbarPlugin Not showing SETTINGS option in Version 2.6

### ISSUE THREAD

**Issue by kunallondhe** [n2cms/n2cms#637]
I have updated N2CMS version from Version: 2.4 File Version: 2.4.12.21 to Version: 2.6 File Version: 2.6.3.8

In My previous Version 2.4, I was able to see "SETTINGS" option in the toolbar and using this when I Save the settings after checking the "Show non-pages in navigation"  checkbox. 

It displays "Setting" and "SiteResource Container" nodes in the Tree, as shown in below screenshot:

![image](https://cloud.githubusercontent.com/assets/108206/7540877/9ee9f4dc-f5ce-11e4-8851-893628f8467a.png)

But, after updating it to Version 2.6, it is not showing "SETTINGS" options in the toolbar. Refer below screenshot:

![image](https://cloud.githubusercontent.com/assets/108206/7541082/20c643e2-f5d0-11e4-89ac-385e70538180.png)

When I try to add "SETTINGS"  option in toolbar explicitly from code then it is showing following error :

![image](https://cloud.githubusercontent.com/assets/108206/7541019/91d35b0c-f5cf-11e4-832e-29cffc2591f3.png)

**Comment by libardo** [comment]
Thé settings have moved to the n2 menu (top left). Can you find it?

/c

--- Originalmeddelande ---

Från: "kunallondhe" <notifications@github.com>
Skickat: 8 maj 2015 18:54
Till: "n2cms/n2cms" <n2cms@noreply.github.com>
Ämne: [n2cms] ToolbarPlugin Not showing SETTINGS option in Version 2.6 (#637)

I have updated N2CMS version from Version: 2.4 File Version: 2.4.12.21 to Version: 2.6 File Version: 2.6.3.8

In My previous Version 2.4, I was able to see "SETTINGS" option in the toolbar and using this when I Save the settings after checking the "Show non-pages in navigation"  checkbox.

It displays "Setting" and "SiteResource Container" nodes in the Tree, as shown in below screenshot:

![image](https://cloud.githubusercontent.com/assets/108206/7540877/9ee9f4dc-f5ce-11e4-8851-893628f8467a.png)

But, after updating it to Version 2.6, it is not showing "SETTINGS" options in the toolbar. Refer below screenshot:

![image](https://cloud.githubusercontent.com/assets/108206/7541082/20c643e2-f5d0-11e4-89ac-385e70538180.png)

When I try to add "SETTINGS"  option in toolbar explicitly from code then it is showing following error :

![image](https://cloud.githubusercontent.com/assets/108206/7541019/91d35b0c-f5cf-11e4-832e-29cffc2591f3.png)

---
Reply to this email directly or view it on GitHub:
https://github.com/n2cms/n2cms/issues/637

**Comment by kunallondhe** [comment]
No,In the left menu of version 2.6 Site Setting representing  "Site" functionality,Refer following screen shot.

![image](https://cloud.githubusercontent.com/assets/108206/7544255/1e2f199c-f5e9-11e4-9bee-6822da1bdbbd.png)

**Comment by libardo** [comment]
Aha! Now I understand.


That feature moved to the right-click-menu (Show parts).


/Cristian





Från: kunallondhe
Skickat: ‎fredag‎ den ‎8‎ ‎maj‎ ‎2015 ‎21‎:‎47
Till: n2cms/n2cms
Kopia: Cristian Libardo





No,In the left menu of version 2.6 Site Setting representing "Site" functionality,Refer following screen shot.

 image

**Comment by kunallondhe** [comment]
It's not working for me. 

i have created country list through setting and used that country in dropdown.  Storing that country value for each page as property. 

now if i want to add more country in list , then it is not possible. previously, it is possible.

is there any other way to display all setting like older version.

**Comment by kunallondhe** [comment]
Please go through with the following findings :

Login to N2 -> Then in new browser tab -> Try to browse the below URL . http://localhost/N2/Content/Navigation/Tree.aspx?selected=%2fstart%2f URL


the above URL, displays the "Setting" and "CMS SiteResource Container" nodes in the Tree, as shown in below screenshot: 

![image](https://cloud.githubusercontent.com/assets/108206/7550760/7f8f08b2-f68b-11e4-82b6-cb83c64c1516.png)

And In N2 -> Edit Page -> Nodes are not present but some time it happen. Maybe it's a cache problem or tree is not refreshing properly. Refer following screen shot :

![image](https://cloud.githubusercontent.com/assets/108206/7550774/61199ffe-f68c-11e4-8a08-4e239b494db0.png) 

<b> Can you please look into this issue asap. it will really helpful. </b>

**Comment by libardo** [comment]
I want to make really sure you found the option indicated by the screenshot.

![image](https://cloud.githubusercontent.com/assets/153667/7556420/492e2034-f76f-11e4-8161-420defe832c7.png)

**Comment by kunallondhe** [comment]
Yes. i am able to see the "Show Parts" option for pages. But by using this option not able to see "Settings".

<b>Consider following example </b>
i have created a page "How to take the perfect landscape photo". In this page "Show Parts" shows all parts but not settings. This page uses "TravelArea Setting"  to populate  TravelAreas in dropdown but "setting" option is not available.

Please refer following screen shot.

![image](https://cloud.githubusercontent.com/assets/108206/7562634/8e57c974-f7f3-11e4-8f6b-cb9f6ebe03ac.png)

In older version, i was able to add new TravelArea through the "SIte Setting" -> "TravelArea Settings" but now in newer version there is no option to add TravelAreas.

**Comment by kunallondhe** [comment]
Please go through with the following findings: 

In the tree, if we change the position of any tree node then tree showing "Setting" and "CMS Site Resource Container" node.

In the below screenshot, I have moved my "Trash" node to the second position then the tree is showing both the node.

Before UI
![image](https://cloud.githubusercontent.com/assets/108206/7615001/2e9c1a32-f9b8-11e4-9b5d-417afda0ec88.png)

After UI
![image](https://cloud.githubusercontent.com/assets/108206/7615082/a892eeb0-f9b8-11e4-86ff-900da7fde8c4.png)

<b>can you please help me out with this issue asap. </b>

**Comment by DejanMilicic** [comment]
@kunallondhe Is this still an issue for you?
